baseapp/baseapp_test.go

```diff
@@ -139,7 +131,7 @@ func setupBaseAppWithSnapshots(t *testing.T, blocks uint, blockTxs int, options
 	snapshotStore, err := snapshots.NewStore(dbm.NewMemDB(), snapshotDir)
 	require.NoError(t, err)
 	teardown := func() {
-		os.RemoveAll(snapshotDir)
+		require.NoError(t, os.RemoveAll(snapshotDir))
```

Comment:
```suggestion
		os.RemoveAll(snapshotDir) //nolint: errcheck
```

Reply:
Why?

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/7674#discussion_r512318354, it would make the test fail for reasons that are unrelated to the code undergoing testing.